go-controller/pkg/config/config.go

```diff
@@ -760,6 +762,11 @@ var OVNGatewayFlags = []cli.Flag{
 		Usage:       "Setup nodeport based ingress on gateways.",
 		Destination: &cliConfig.Gateway.NodeportEnable,
 	},
+	&cli.BoolFlag{
+		Name:        "disable-snat-external-gws",
+		Usage:       "Setup SNAT traffic on gateways.",
```

Comment:
usage description is backwards.

Also, is this necessary? We should be moving toward having things only be configured via the config file.

Reply:
unsure, was just cargo-culting to keep consistency